unicodetools/data/ucd/dev/Index.txt

```diff
@@ -1603,6 +1603,7 @@ direct sum	2295
 Directional Format Characters	202A
 DIRECTIONAL FORMATTING, POP	202C
 DIRECTIONAL ISOLATE, POP	2069
+DIRHAM SIGN, UAE	20C3
```

Comment:
@Ken-Whistler usually adds things into the Index late in a release. Ken, do you want this here in the pending-for-18 pull request?